bundles/org.openhab.binding.velbus/src/main/java/org/openhab/binding/velbus/internal/handler/VelbusSensorHandler.java

```diff
@@ -89,12 +94,44 @@ public void handleCommand(ChannelUID channelUID, Command command) {
             byte[] packetBytes = packet.getBytes();
             velbusBridgeHandler.sendPacket(packetBytes);
         }
+
+        if (isButtonChannel(channelUID) && command instanceof StringType) {
+            StringType stringTypeCommand = (StringType) command;
+
+            if (stringTypeCommand.equals(PRESSED) || stringTypeCommand.equals(LONG_PRESSED)) {
+                VelbusButtonPacket packet = new VelbusButtonPacket(getModuleAddress().getChannelIdentifier(channelUID));
+
+                try {
+                    packet.Pressed();
+                    velbusBridgeHandler.sendPacket(packet.getBytes());
+                    Thread.sleep(20);
```

Comment:
I don't think you are allowed to let the thread sleep here.
But you could use the scheduler, like [here](https://github.com/openhab/openhab-addons/blob/5ec535f37bef6f2491da64cd884af02b454e7d03/bundles/org.openhab.binding.velbus/src/main/java/org/openhab/binding/velbus/internal/handler/VelbusBridgeHandler.java#L155)

Reply:
I d'idn't check this part of your code, the 20 ms delay is useless then. I've remove this from my code.